src/DroneSafetyApi/Models/HeatMapPoint.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace DroneSafetyApi.Models
+{
+    public class HeatMapPoint
+    {
+        public double x { get; set; }
```

Comment:
I think the Caesium heat map uses x and y but will confirm with Bence. If so it just avoids an extra processing step on the front end.